src/discussions/common/AuthorLabel.jsx

```diff
@@ -41,6 +42,10 @@ function AuthorLabel({
 
   const className = classNames('d-flex align-items-center', labelColor);
 
+  const showUserNameAsLink = useShowLearnersTab()
+      && linkToProfile && author
+      && learnersTabEnabled && author !== messages.anonymous;
+
```

Comment:
You already checking the `learnersTabEnabled` in `useShowLearnersTab()`. So you can remove it here. 
nit: 
`const showUserNameAsLink = useShowLearnersTab() && linkToProfile && author 
&& author !== messages.anonymous;`

Reply:
done